### DISCUSSION TITLE
Créer un fichier docker-compose.yml pour configuré l'environnement de déployement du projet

### DISCUSSION THREAD

**Discussion by StevyMarlino** [laravelcm/jobs-portail-issues#3] [Ideas]
Ici je parle d'avoir un docker qui aura déjà les images des services utilisé dans le projet suivant les configuration de l'environnement de déployment et plus...

**Comment by mckenziearts** [comment]
Le projet est automatiquement déployé en utilisant Laravel Forge du coup pour ce projet Docker n'est pas vraiment nécessaire vu que tous les deploy sont automatique.
